src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -253,6 +259,9 @@ public void validateInternalCollaborators(DataAccessRequest payload, User reques
    * @return The updated DataAccessRequest
    */
   public DataAccessRequest updateByReferenceId(User user, DataAccessRequest dar) {
+    if (dar.getDraft() != true) {
```

Comment:
```suggestion
    if (!dar.getDraft()) {
```

Alternatively, you could add a method `isReadOnly()` to `DataAccessRequest` as an alias for `!draft` to make this read better.

Reply:
I'm going to hold off on changing this until 1520 lands because there's a chance that getDraft can return null.  That will resolve in that branch and then I can leverage this suggestion.